config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-end
+  resources :users#, only: [:new, :create, :edit, :update,:show]
```

Comment:
Please use the routes specified by The Odin Project here: 
```
 resources :users, only: [:new, :create, :edit, :update]
```
![image](https://user-images.githubusercontent.com/2739245/111880290-31d02880-8970-11eb-8361-0e45bb3bea6c.png)

![image](https://user-images.githubusercontent.com/2739245/111880274-136a2d00-8970-11eb-81e6-c4700c2b05f0.png)
https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms

Reply:
thank a lot for your review I'm done with required changes